range_sensor_broadcaster/test/test_range_sensor_broadcaster.cpp

```diff
@@ -141,6 +150,22 @@ TEST_F(RangeSensorBroadcasterTest, Activate_RangeBroadcaster_Success)
   ASSERT_EQ(
     range_broadcaster_->on_activate(rclcpp_lifecycle::State()),
     controller_interface::CallbackReturn::SUCCESS);
+
+  // check interface configuration
+  auto cmd_if_conf = range_broadcaster_->command_interface_configuration();
+  ASSERT_THAT(cmd_if_conf.names, IsEmpty());
+  auto state_if_conf = range_broadcaster_->state_interface_configuration();
+  ASSERT_THAT(state_if_conf.names, SizeIs(1lu));
```

Comment:
Maybe in the future it also makes sense to check for the type to be NONE. What do you think @christophfroehlich ?

Reply:
I'm not sure about the implications, where this information is used etc. But yes, why not

Reply:
I think you can have type as ALL and not specify any names. So, the controller manager gets you access to all interfaces. That's the case for joint state broadcaster when no joints are specified I guess. Well this is for future, for now the PR looks good to me